aiohttp_session/redis_storage.py

```diff
@@ -1,14 +1,14 @@
 import json
 import uuid
-from distutils.version import StrictVersion
 from typing import Any, Callable, Optional
 
 from aiohttp import web
 
 from . import AbstractStorage, Session
 
 try:
-    import aioredis
+    import redis
```

Comment:
I'd consider just doing `from redis import __version__ as redis_version`. Which will also avoid the lgtm warning.